### PR TITLE
Fix Bicep versioning

### PR DESCRIPTION
**Pull request by AaronCrawfis**
Fixes Bicep versioning for PRs

### CODE REVIEW

File: .github/scripts/get_docs_version.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+gitRefName = os.getenv("GITHUB_REF_NAME")
+
+with open(os.getenv("GITHUB_ENV"), "a") as githubEnv:
+    version = "DOCS_CHANNEL=edge"
+
+    if gitRefName is None:
+        print("This is not running in github, GITHUB_REF_NAME is null. Assuming a local build. Setting DOCS_CHANNEL to 'edge'")
+        version = "DOCS_CHANNEL=edge"
+    elif gitRefName == 'edge':
```
Comment (ytimocin) [inline .github/scripts/get_docs_version.py]: Should this be case-sensitive for any case?

File: .github/scripts/get_docs_version.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        print("This is an edge build, setting DOCS_CHANNEL to 'edge'")
+        version = "DOCS_CHANNEL=edge"
+    elif gitRefName.startswith('v'):
+        print("This is a release build, setting DOCS_CHANNEL to verion")
```
Comment (ytimocin) [inline .github/scripts/get_docs_version.py]:
```suggestion
        print("This is a release build, setting DOCS_CHANNEL to version")
```

File: .github/scripts/get_docs_version.py (hunk 1 above)
Comment (AaronCrawfis) [reply .github/scripts/get_docs_version.py]: The format we expect is `edge`, which should match `GIT_BASE_REF`. There shouldn't be any cases where the cases differ as far as I know

Comment (willtsai) [reply .github/scripts/get_docs_version.py]: perhaps making it case insensitive may proof this against what-ifs?

```suggestion
    elif gitRefName.lower() == 'edge':
```

**Review by willtsai — COMMENTED** [review]
pulled and built this branch locally, looks ok with major issues to me. will take a look again after you've responded to my feedback.

**Review by willtsai — APPROVED** [review]

**Pull request merged**